src/core/autofill.cc

```diff
@@ -219,8 +221,17 @@ AutoFill::Create(
   // Unable to determine the platform so just use the simple autofill,
   // or null if that fails.
   {
+    if (platform == Platform::PLATFORM_UNKNOWN) {
+      LOG_WARNING << "Autofiller failed to detect the platform (verify "
+                     "contents of version directory or use --log-verbose=1 for "
```

Comment:
Shouldn't this be model directory instead of version directory? Also what happens when there is an extra file in the same directory as the version folders and config.pbtxt (say a .gitkeep)

Reply:
Fixed. Ignoring hidden files as well.